external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaSourceOffset.scala

```diff
@@ -27,13 +28,21 @@ import org.apache.spark.sql.execution.streaming.Offset
  */
 private[kafka010]
 case class KafkaSourceOffset(partitionToOffsets: Map[TopicPartition, Long]) extends Offset {
+  import org.json4s.jackson.Serialization.{write}
```

Comment:
Actually its easier to read the code later if you import Serialization, and then use Serialization.write. Same for read.